nginx_stage/lib/nginx_stage/socket_file.rb

```diff
@@ -27,9 +29,8 @@ def socket
     # Delete the socket from the file system
     def delete
       @socket.delete
-    rescue
-      $stderr.puts "Unable to delete socket file at #{socket}"
+    rescue StandardError
```

Comment:
I don't think file Errors are `StandardError`s so I can't pull this in as is. I.e., this wouldn't actually get rescued.

Reply:
Actually I'm wrong. This catches file errors.